src/components/v2/values/common/ChartValuesSelectors.tsx

```diff
@@ -223,6 +220,11 @@ export const ChartRepoSelector = ({
                             MenuList: customMenuListItem,
                         }}
                         styles={{
+                            menuList: (base) => ({
```

Comment:
is this change intentional? (not related to this story though)

Reply:
Yes.